ee/modules/502-delivery/openapi/config-values.yaml

```diff
@@ -0,0 +1,126 @@
+type: object
+required:
+  - auth
+properties:
+  argocd:
+    type: object
+    default: {}
+    description: "ArgoCD configuration."
+    properties:
+      admin:
+        type: object
+        default: {}
+        properties:
+          enabled:
+            type: boolean
+            default: false
+            description: "Enable ArgoCD admin user. Might be helpful when Dex is not used for authentication."
```

Comment:
Can we move admin password closer to this option? If I get it right, now if I want to login as admin user with my password, I need to arrange settings like the following:
```
auth:
  password: 12345
argocd:
  admin:
    enabled: true
```

Reply:
Argo CD creates admin password and stores it in secret `argocd-initial-admin-secret`. So, user can save it and delete the secret, to hide it.

Secret `argocd-secret` contains the password hash and modification time

- `admin.password`
- `admin.passwordMtime`

We can use this flow to re-generate the password and store it in the Argo-way. Supposedly, on next iteration.

Should we re-invent the way Argo CD manages the password?  I'd remove `auth` field from the settings at all, as we can rely on Argo CD authentication instead of nginx basic auth.

Reply:
Agreed on removing auth section

Reply:
Removed